From ced176f16f23efed0c922bfe92f9422890942073 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Sun, 11 Oct 2015 09:02:26 -0700
Subject: [PATCH 2006/2012] MacPorts Only: Fix regression introduced when
 fixing PR24776

Partially reverts commit efecb2c285bd444b6def43ac62e5f0278df387eb

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
---
 make/platform/clang_darwin.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git llvm_release_38/projects/compiler-rt/make/platform/clang_darwin.mk macports_release_38/projects/compiler-rt/make/platform/clang_darwin.mk
index 626e747..3f5e948 100644
--- llvm_release_38/projects/compiler-rt/make/platform/clang_darwin.mk
+++ macports_release_38/projects/compiler-rt/make/platform/clang_darwin.mk
@@ -18,7 +18,7 @@ CheckArches = \
     if [ "X$(3)" != X ]; then \
       for arch in $(1); do \
         if ( $(LD) -v 2>&1 | grep "configured to support" | tr ' ' '\n' | grep "^$$arch$$" >/dev/null 2>/dev/null ) || (! ( $(LD) -v 2>&1 | grep -q "configured to support" ) && test "$$arch" = "i386" -o "$$arch" = "x86_64" -o "$$arch" = "ppc" -o "$$arch" = "ppc64" ); then \
-          if $(CC) -arch $$arch \
+          if $(CC) -arch $$arch -c \
             -integrated-as \
             $(ProjSrcRoot)/make/platform/clang_darwin_test_input.c \
             -isysroot $(3) \
-- 
2.10.1

